packages/contract-wrappers/src/types.ts

```diff
@@ -225,3 +225,61 @@ export interface DutchAuctionData {
     beginTimeSeconds: BigNumber;
     beginAmount: BigNumber;
 }
+
+export interface CoordinatorServerApprovalResponse {
+    signatures: string[];
+    expirationTimeSeconds: BigNumber[];
+}
+export interface CoordinatorServerApprovalRawResponse {
+    signatures: string[];
+    expirationTimeSeconds: BigNumber;
+}
+
+export interface CoordinatorServerCancellationResponse {
+    outstandingFillSignatures: [
+        {
+            orderHash: string;
+            approvalSignatures: string[];
+            expirationTimeSeconds: BigNumber;
+            takerAssetFillAmount: BigNumber;
+        }
+    ];
+    cancellationSignatures: string[];
+}
+
+export interface CoordinatorServerResponse {
+    isError: boolean;
+    status: number;
+    body?: CoordinatorServerCancellationResponse | CoordinatorServerApprovalRawResponse;
+    error?: any;
+    request: {
+        signedTransaction: SignedZeroExTransaction;
+        txOrigin: string;
+    };
+    coordinatorOperator: string;
+    orders?: Array<SignedOrder | Order>;
+}
+
+export class CoordinatorServerError extends Error {
```

Comment:
I wouldn't consider classes as types (as understood in Typescript). Let's move this to it's own file.